src/main/java/kr/codesquad/library/admin/service/BookAdminService.java

```diff
@@ -0,0 +1,141 @@
+import kr.codesquad.library.domain.category.Category;
+import kr.codesquad.library.global.config.properties.InterparkProperties;
+import kr.codesquad.library.global.error.exception.domain.BookNotFoundException;
+import kr.codesquad.library.global.error.exception.domain.BookcaseNotFoundException;
+import kr.codesquad.library.global.error.exception.domain.CategoryNotFoundException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static kr.codesquad.library.admin.common.ConstantsCoveringMagicNumber.ADMIN_PAGE_SIZE;
+import static kr.codesquad.library.admin.common.ConstantsCoveringMagicNumber.MINIMUM_PAGE_NUMBER;
+
+@RequiredArgsConstructor
+@Slf4j
+@Service
+@Transactional(readOnly = true)
+public class BookAdminService {
+
+    private final BookAdminRepository bookAdminRepository;
+    private final CategoryAdminRepository categoryAdminRepository;
+    private final BookcaseAdminRepository bookcaseAdminRepository;
+    private final InterparkProperties interparkProperties;
+
+    public BooksWithPagingResponse findAllBooks(int page) {
+        Page<Book> books = bookAdminRepository.findAllWithCategory(PageRequest.of(validatePageNumber(page), ADMIN_PAGE_SIZE));
+        List<Book> bookEntities = books.getContent();
+        List<BookSummary> bookSummaries = bookEntities.stream()
+                                                      .map(BookSummary::from)
+                                                      .collect((Collectors.toList()));
+        PagingProperties pagingProperties = PagingProperties.from(books);
+        return BooksWithPagingResponse.of(bookSummaries, pagingProperties);
+    }
+
+
+    @Transactional
+    public Long createNewBook(BookFormRequest bookFormRequest) {
+        Category category = categoryAdminRepository.findById(bookFormRequest.getCategoryId())
+                                                   .orElseThrow(CategoryNotFoundException::new);
+        Bookcase bookcase = bookcaseAdminRepository.findById(bookFormRequest.getBookcaseId())
+                                                   .orElseThrow(BookcaseNotFoundException::new);
+        Book newBook = bookAdminRepository.save(Book.of(bookFormRequest, category, bookcase));
+        log.debug("New Book ID ::: {}", newBook.getId());
```

Comment:
현재 로그가 debug로 설정되어있습니다. log를 사용해서 제대로 됐는지 확인했다면 삭제해주세요~

Reply:
이건 제 실수네요. 수정 하겠습니다.

Reply:
`Nit`

```suggestion
        log.debug("### New Book ID ::: {}", newBook.getId());
```
이렇게 하면 ctrl+f 의 검색어를 `New Book ID` 가 아닌 `###` 으로 해서 찾을 수 있어 선호합니다.